src/Command/Pull/PullCommandBase.php

```diff
@@ -244,6 +249,11 @@ protected function downloadDatabaseDump(
   ): string {
     $database_backups = new DatabaseBackups($acquia_cloud_client);
     $backups_response = $database_backups->getAll($environment->uuid, $database->name);
+    if (!count($backups_response)) {
```

Comment:
It seems like we may want to check how old this db backup is. For instance, if it's 6 months old, we probably want to create a new backup. What do you think about setting a maximum age?

Reply:
I don't like the idea of automatically creating an on-demand backup based on an arbitrary cutoff. I'll change it to print a warning though.